source/standards/configuration-management.html.md.erb

```diff
@@ -0,0 +1,25 @@
+---
+title: Use configuration management
+expires: 2018-05-16
+---
+
+# <%= current_page.data.title %>
+
+<%= partial :expires %>
+
+You should use configuration management to track and control changes in large software deployments. When using configuration management you ensure your environment configuration is documented and [stored as code][].
+
+For example, when you run virtual machines (VM) in [Amazon Elastic Compute Cloud (Amazon EC2)][] and the VMs instance configuration becomes too complex to manage either during its deployment or within your [Amazon Machine Image (AMI)][] use configuration management.
```

Comment:
We could do with saying why your configuration would become too complex, and under what circumstances. What do we mean when we say something's 'complex' in this context?